README.md

```diff
@@ -379,6 +381,8 @@ With bfg the files that exist on your latest commit will not be affected. For ex
 
 Note, if you renamed your file as part of a commit, e.g. if it started as `LargeFileFirstName.mp4` and a commit changed it to `LargeFileSecondName.mp4`, running `java -jar ~/Downloads/bfg.jar --delete-files LargeFileSecondName.mp4` will not remove it from git history. Either run the `--delete-files` command with both filenames, or with a matching pattern. As explained above, any files present in the repo on your latest commit will be safe.
 
+#### Pushing your chnaged repo history
```

Comment:
Spelling

Reply:
Darn, sorry, in the middle of work an inadverdently updated the pull request. Will notify here once it is completed.